virtualenvironment/bin/Activate.ps1

```diff
@@ -0,0 +1,241 @@
+<#

```

Comment:
folder `virtualenvironment` should be added to a `.gitignore` file . Virtual environments should not be uploaded to the repo. Virtualenv creation should be documented in the Readme of the repo